def/e4x.ts

```diff
@@ -83,4 +87,4 @@ module.exports = function (fork) {
         .bases("XML")
         .field("target", String)
         .field("contents", or(String, null));
```

Comment:
This file is indented with 4 spaces, whereas most are indented with 2. Have you thought about enabling prettier for this repo?

Reply:
I've been slowly shifting to 2 spaces per tab as I make changes to files, and I haven't touched this one in a while. Now seems like a fine time to reindent this code!

You're welcome to use any formatting tool you like when working on a PR, but I'd rather not add Prettier to the workflow for this repository. The readability of `def(...).bases(...).field(...)` expressions often benefits from idiomatic line breaks and indentation, I've found, and auto-formatting tools would only interfere.